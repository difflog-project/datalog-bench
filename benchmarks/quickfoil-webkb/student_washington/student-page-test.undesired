2889
2580
2648
2486
2732
3220
2795
3183
2285
2951
988
996
941
2417
2612
3076
2284
2588
2645
3179
2871
2877
2571
2755
2629
2699
3075
2633
2874
2586
3025
2993
3040
3032
3012
3219
2668
2990
2439
2979
2452
2555
2980
802
2946
2658
3146
975
760
2723
2901
2791
3046
3121
3011
3136
986
2366
2955
2727
945
2748
2286
2894
2448
3007
2824
2779
2407
2969
2314
2295
2638
2792
950
2483
2705
2726
2502
3029
2568
2348
2585
568
2394
2574
2968
599
2329
2922
2557
2692
2402
937
2812
2681
2351
2814
2949
2611
2479
2338
2841
805
2703
2998
3027
2978
2734
712
2335
2689
2974
973
2987
2444
684
3132
3184
3010
3072
2767
2445
2595
2866
2298
569
2984
2385
3138
2756
2477
3051
2368
2837
3047
2712
3005
3008
2956
3158
3036
2578
709
2762
2747
3112
2654
2989
3087
2337
3151
2466
2331
3214
927
2813
3088
3140
703
3038
978
3139
2663
2352
981
934
2850
2294
2297
3202
2914
3033
3190
2453
2545
2741
2283
2582
3115
2640
2676
3152
714
968
949
2378
2691
933
689
2567
2853
2442
2584
2724
2693
3091
3161
3205
3118
2669
2465
804
2589
2467
2975
2803
3065
977
2807
2961
2533
2464
2952
985
3212
2492
2406
3122
3195
3016
2936
710
3211
2639
2941
2349
3181
944
2683
2420
3001
2855
598
3189
2680
2577
3113
2591
3156
979
2818
2554
3048
3056
2868
3120
2926
2778
2537
3206
2522
2744
2496
2497
2768
2428
800
971
2902
2701
688
3006
3166
2365
3160
2924
2336
2749
3071
2386
695
2933
2508
2475
2413
2785
3034
685
2383
3070
3157
3131
2494
2862
2600
2958
2472
2667
2953
2432
3130
2535
2917
2333
3041
2809
2434
2424
2656
3109
2401
2461
2316
982
2463
2916
2505
2370
2334
3106
3144
2811
2907
2753
2643
2939
2761
2720
2931
2790
2519
2690
2948
3035
2897
3083
948
2350
990
2882
2380
3074
2981
2490
2661
2686
2992
2672
2523
2487
2842
597
2387
3114
3167
2750
936
2392
791
2312
2704
2482
2959
959
2834
2839
795
987
2930
3210
2905
2994
2478
2666
2307
2462
3030
2893
2986
2962
951
2707
3018
2923
2460
2499
2742
2906
2815
2615
3082
3037
2371
2355
3147
2863
2746
3135
706
928
2372
2982
3003
963
3177
3141
2636
2527
1003
3124
2296
2758
3105
2559
793
3053
2403
939
2783
2682
2714
2498
2602
2937
2469
704
701
3145
3086
2393
972
2801
2455
3110
2771
2610
2684
2426
2340
2379
2696
3170
2609
2536
2620
2302
2856
967
2966
3079
1001
2764
2869
3004
2976
2510
2825
2474
2518
2471
2798
983
3044
2635
3209
2845
3015
2564
2456
2468
2504
2524
3067
2300
938
3024
3000
2819
3180
2616
2657
2782
2293
2396
2977
3196
3096
3154
2572
2408
3084
2641
2500
2674
2940
2281
694
2909
2808
2957
2601
2892
2943
2501
960
713
3060
2427
2328
2867
2854
2800
686
2934
2911
3194
698
2576
2695
2489
3095
707
2804
2541
3169
2306
2449
956
2528
2520
2822
2759
2885
2356
930
2481
2910
798
3077
2794
2423
2754
2802
2476
2288
2346
2558
687
2363
2757
3182
3104
3050
2799
3123
2597
806
2786
2721
2823
2411
3178
691
2322
796
708
2776
974
2960
2621
3197
2425
2604
2888
3200
2561
2781
999
2290
2653
3133
2618
961
2451
2587
3153
2614
2305
2435
2343
2925
2872
2710
697
2919
2708
801
2659
3125
2605
2903
2634
2743
2827
2929
2826
2367
3042
3119
2347
2844
2543
2342
2651
2895
2865
2430
2846
2433
2623
2898
2529
2525
929
2655
2470
3039
2649
2765
2878
2729
2851
2598
3022
3026
3207
2400
2642
992
2405
3066
2361
2311
3094
600
3002
2716
2964
2317
3117
2516
2454
2458
3017
2303
2890
2507
2817
2876
2315
2763
2550
3188
2738
952
2548
2354
2628
2369
2985
958
980
3199
2534
2728
2594
3043
3204
3193
2289
3143
2388
3176
790
3174
2631
690
3078
3102
3085
3093
2632
2446
2806
970
2797
2443
2752
2438
2875
955
2775
3185
2725
2570
2915
2596
2774
3150
3063
3107
2679
946
2299
2457
2847
3137
3165
1000
2607
2530
2733
570
2736
2404
2416
942
3054
2362
2921
3090
995
2900
966
2539
2920
2730
2789
2713
2447
2694
3089
3201
3203
2397
2389
3162
2553
3213
2838
2485
2514
2887
3111
3108
2546
2650
2552
3073
2918
2563
2731
3218
3049
953
2954
2318
3168
2326
692
2542
2341
2327
962
2718
943
932
2373
2745
991
2665
2590
2852
2415
2353
2581
2859
2849
3021
2678
2788
2503
2702
693
2647
2829
2551
2617
3198
2511
792
2860
3081
3127
3068
2599
2324
2793
2833
2357
2391
2330
2583
2544
2971
2622
984
3100
2319
3064
3186
2796
2375
2821
2512
3057
3116
797
2282
2608
2381
2309
3009
989
940
3020
3208
2323
787
2436
3055
2441
2320
3134
2419
2619
3058
3128
2671
3045
2440
2627
3172
2376
2848
2928
2531
2521
957
3164
2913
3159
2304
2912
2652
2715
2431
2945
2573
2579
2965
2662
3163
3080
2625
969
2997
2873
3191
3103
595
2835
2540
935
3101
2861
2995
2480
2569
3171
3069
2787
2881
3192
3061
2697
3217
2660
2700
2772
2991
2292
2688
2613
3031
3028
2840
788
2532
2972
2310
2484
2513
2988
976
2879
2562
2932
2377
997
2313
2344
705
2935
2810
2399
2927
2547
2560
3173
2766
3092
2332
2820
998
3175
994
2515
2624
2884
965
2421
2999
2938
2606
2973
2592
700
3023
2773
2687
2864
2364
2409
2603
3098
2549
2685
2437
2384
2722
2565
3019
2308
3216
2739
2646
567
799
3215
2670
2740
2538
2644
2410
2673
1002
2983
2963
3097
596
2831
2488
2422
2301
3187
696
2780
2506
2908
2339
3148
2828
2760
2418
2942
699
2886
2321
2777
2509
2870
2395
993
2709
3052
2737
954
2291
2637
2717
2970
2950
2412
2858
2664
3149
2491
2630
2832
2711
3014
2495
2675
2843
2360
2836
2751
2517
3126
2626
2677
2575
2784
3062
3155
947
2769
2967
786
2896
2593
2944
2414
2566
2358
711
964
2287
2770
2883
2830
794
3142
2947
2382
2459
2719
2359
3099
2374
3013
2493
2429
2698
2816
2891
3059
2450
931
2899
2904
2735
3129
803
789
2325
2857
2880
2706
2398
2996
2556
702
2526
2390
2805
2473
2345
