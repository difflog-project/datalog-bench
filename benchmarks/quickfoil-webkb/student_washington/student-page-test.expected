258
254
185
245
226
234
162
227
166
259
211
253
221
219
189
210
241
278
218
242
261
181
249
268
225
267
156
230
224
205
170
232
196
160
251
157
250
174
248
163
176
274
191
229
208
201
168
273
200
222
246
255
207
187
276
279
158
214
280
266
272
194
263
216
199
213
202
193
264
236
231
277
257
244
186
198
203
173
197
167
252
182
235
281
206
183
179
265
247
256
260
269
275
195
262
164
270
169
159
238
223
177
243
171
161
188
215
180
192
172
240
190
175
184
217
212
220
165
178
239
237
233
228
209
204
271
